pkgs/top-level/python-packages.nix

```diff
@@ -4321,6 +4322,7 @@ in {
     stdenv = if stdenv.isDarwin then pkgs.clangStdenv else pkgs.stdenv;
     inherit (pkgs.darwin.apple_sdk.frameworks) Cocoa;
     inherit (pkgs) pkgconfig;
+    inherit (pkgs.qt5) wrapQtAppsHook;
```

Comment:
@rnhmjoj according to my tests, this line is what's causing ofborg's eval error and it causes this:

```
env NIX_PATH=nixpkgs=$PWD nix-shell -p 'pythonPackages.matplotlib.override {enableGtk3=true;}' --run 'python -c "import matplotlib.pyplot as plt; plt.plot([1,2,3],[4,5,6]); plt.show()"'
```

(The command that reproduces https://github.com/NixOS/nixpkgs/issues/25351 ) to fail in the same way.

Reply:
Oh, I see now: it's the python 2 version! I'll fix it by I guess it will be removed soon enough.